tests/test_fetcher.py

```diff
@@ -0,0 +1,146 @@
+#!/usr/bin/env python
+
+"""
+<Program>
+  test_download.py
+
+<Author>
+  Teodora Sechkova tsechkova@vmware.com
+
+<Started>
+  December 18, 2020.
+
+<Copyright>
+  See LICENSE-MIT OR LICENSE for licensing information.
+
+<Purpose>
+  Unit test for 'fetcher.py'.
```

Comment:
Specifically unit tests for requests_fetcher?

Reply:
Fixed in be399d2.